### ISSUE TITLE
Ability to define common env variable for all steps of a task

### ISSUE THREAD

**Issue by apoorvam** [leopardslab/dunner#92]
### Approach 1
Have a global `envs` and `mounts` section applicable for all tasks

Example:

```yaml
envs:
  - FOO=bar
  - HELLO=world

mounts: 
  - /var:/foo

tasks:
  deploy:
  - image: 'emeraldsquad/sonar-scanner'
    commands:
    - ['sonar', 'scan']
  - image: 'golang'
    commands:
    - ['go', 'install']
  - image: 'mesosphere/aws-cli'
    commands:
    - ['aws', 'elasticbeanstalk update-application --application-name myapp']
    envs:
     - AWS_ACCESS_KEY_ID=`$AWS_KEY`
     - AWS_SECRET_ACCESS_KEY=`$AWS_SECRET`
     - AWS_DEFAULT_REGION=us-east1
  ```

**Comment by agentmilindu** [comment]
Capturing in #146

**Comment by ayushjn20** [comment]
Closed with reference to #148
